src/components/structures/SpaceHierarchy.tsx

```diff
@@ -374,12 +374,12 @@ export const showRoom = (cli: MatrixClient, hierarchy: RoomHierarchy, roomId: st
         action: Action.ViewRoom,
         should_peek: true,
         room_alias: roomAlias,
-        room_id: room.room_id,
+        room_id: roomId,
```

Comment:
This has changed from using the variable declared on line 360 to using one of the arguments. Probably intentional, just raising it in case this would change the behaviour here.

Reply:
I mean we never assert that `room.room_id === roomId` but if they are not the same (assuming a room object is non-null) then the app and/or server are *really* broken. We make this assumption in many places.